packages/parser/README.md

```diff
@@ -0,0 +1,27 @@
+<p>
+  <img width="100%" src="" alt="Solid Primitives parser">
+</p>
+
+# @solid-primitives/parser
```

Comment:
`jsx-parser`?
Maybe `jsx` doesn't make sense if you put on the "well actually" glasses, but `parser` is wayyy too general

Reply:
good by me. 

changed 
`parser` to `jsx-parser`, 
`createParser` to `createJSXParser` and
`tokenize` to `createToken`